webapp/pages/index.vue

```diff
@@ -75,7 +75,10 @@ export default {
     MasonryGridItem,
   },
   data() {
-    const { hashtag = null } = this.$route.query
+    let { hashtag = null } = this.$route.query
+    if (hashtag) {
+      hashtag = decodeURI(hashtag)
```

Comment:
Weird, the hashtag is not automatically decoded? :confused:

Reply:
May I add to much `decodeURI` …